### DISCUSSION TITLE
Limit the collections that are selectable in the UploadFeature of Lexical Editor

### DISCUSSION THREAD

**Discussion by psusmars** [payloadcms/payload#7442] [Feature Requests & Ideas]
Right now, it seems that there is no way to manage what collections are available in the drawer dropdown when inserting an upload into a rich text input using the UploadFeature. I dove around in the code, I could see this as a really easy addition that could be valuable for the multitenant where there might be multiple upload collections but a user might not want to see them all... Or even have access to them all.

**Comment by zdqcsty** [comment]
Are there any good solutions?  i think is a very important  question
